[{"ContainerToolsException.Message":"Docker command failed with exit code 1.\r\nGet https://mcr.microsoft.com/v2/: dial tcp: lookup mcr.microsoft.com on 192.168.65.1:53: no such host","ContainerToolsException.ErrorCode":"CTC1014","ContainerToolsException.StackTrace":"   at Microsoft.VisualStudio.Containers.Tools.Common.Client.DockerClient.<BuildImageAsync>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<RebuildImageAsync>d__11.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchForDockerfileAsync>d__7.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildAndLaunch.<ExecuteCoreAsync>d__160.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildTask.<ExecuteAsync>d__24.MoveNext()","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.FaultingUserFile.FilePath":"dockerfile","ContainerToolsException.FaultingUserFile.LineStart":"","ContainerToolsException.FaultingUserFile.ColumnStart":"","ContainerToolsException.FaultingUserFile.LineEnd":"","ContainerToolsException.FaultingUserFile.ColumnEnd":"","ContainerToolsException.CommandLine.Program":"docker","ContainerToolsException.CommandLine.TelemetrySafeArguments":"build","ContainerToolsException.CommandLine.ExitCode":"1","ContainerToolsException.CommandLine.ErrorOutput":"Get https://mcr.microsoft.com/v2/: dial tcp: lookup mcr.microsoft.com on 192.168.65.1:53: no such host"},{"ContainerToolsException.Message":"Docker command failed with exit code 1.\r\nInvalid containerPort: 80,8761","ContainerToolsException.ErrorCode":"CTC1014","ContainerToolsException.StackTrace":"   at Microsoft.VisualStudio.Containers.Tools.Common.Client.DockerClient.<BuildImageAsync>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<RebuildImageAsync>d__11.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchForDockerfileAsync>d__7.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildAndLaunch.<ExecuteCoreAsync>d__160.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildTask.<ExecuteAsync>d__24.MoveNext()","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.FaultingUserFile.FilePath":"Dockerfile","ContainerToolsException.FaultingUserFile.LineStart":"","ContainerToolsException.FaultingUserFile.ColumnStart":"","ContainerToolsException.FaultingUserFile.LineEnd":"","ContainerToolsException.FaultingUserFile.ColumnEnd":"","ContainerToolsException.CommandLine.Program":"docker","ContainerToolsException.CommandLine.TelemetrySafeArguments":"build","ContainerToolsException.CommandLine.ExitCode":"1","ContainerToolsException.CommandLine.ErrorOutput":"Invalid containerPort: 80,8761"},{"ContainerToolsException.Message":"Docker command failed with exit code 1.\r\nInvalid containerPort: 80,8761","ContainerToolsException.ErrorCode":"CTC1014","ContainerToolsException.StackTrace":"   at Microsoft.VisualStudio.Containers.Tools.Common.Client.DockerClient.<BuildImageAsync>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<RebuildImageAsync>d__11.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchForDockerfileAsync>d__7.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildAndLaunch.<ExecuteCoreAsync>d__160.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildTask.<ExecuteAsync>d__24.MoveNext()","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.FaultingUserFile.FilePath":"Dockerfile","ContainerToolsException.FaultingUserFile.LineStart":"","ContainerToolsException.FaultingUserFile.ColumnStart":"","ContainerToolsException.FaultingUserFile.LineEnd":"","ContainerToolsException.FaultingUserFile.ColumnEnd":"","ContainerToolsException.CommandLine.Program":"docker","ContainerToolsException.CommandLine.TelemetrySafeArguments":"build","ContainerToolsException.CommandLine.ExitCode":"1","ContainerToolsException.CommandLine.ErrorOutput":"Invalid containerPort: 80,8761"}]